modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -47,9 +47,12 @@ topk(10,count by (job)({__name__=~".+"}))
 
 ** *If the metrics relate to a core {product-title} project*, create a Red Hat support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
 
-. Check the TSDB status in the Prometheus UI.
-.. In the *Administrator* perspective, navigate to *Networking* -> *Routes*.
-.. Select the `openshift-monitoring` project in the *Project* list.
-.. Select the URL in the `prometheus-k8s` row to open the login page for the Prometheus UI.
-.. Choose *Log in with OpenShift* to log in using your {product-title} credentials.
-.. In the Prometheus UI, navigate to *Status* -> *TSDB Status*.
+. Review the TSDB status using the Prometheus web API by running the following commands as a cluster administrator:
```

Comment:
Prometheus web API
->
Prometheus API

Reply:
changed per @simonpasquier to `Prometheus HTTP API`